modules/health_quest/app/serializers/health_quest/v0/pgd_questionnaire_serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module V0
+    class PGDQuestionnaireSerializer
+      include FastJsonapi::ObjectSerializer
+
+      set_id do |object|
```

Comment:
the id in FastJonapi use the `:id` key by default, so the `set_id` should be unnecessary here